src/store/main-store/MainMutations.ts

```diff
@@ -13,4 +13,8 @@ export default class MainMutations extends Mutations<MainState> {
     public changeLanguajeToEnglish(): void {
         this.state.currentLanguage = 'en';
     }
+
+    public activeMaintenancePage(): void {
+        this.state.maintenancePageStatus = !this.state.maintenancePageStatus;
```

Comment:
@gonzalo-plaza Since you want to say "Activate" the correct way may be just to put 'true'